src/components/templates/Blog/postStyles.js

```diff
@@ -1,31 +1,186 @@
+  .Ministries__Contact__Image-Circle {
+    width: 61px;
+    height: 62px;
+
+    @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+      width: 61px;
+      height: 61px;
+    }
+  }
+  .Article__Contact__Position {
+    font-size: 12px !important;
+    color: ${(props) => props.theme.colors.secondary};
+    font-family: ${(props) => props.theme.fontFamily.secondary} !important;
+    text-transform: uppercase;
+  }
+
+  .Article__Contact__Name {
+    font-size: 12px !important;
+    font-family: ${(props) => props.theme.fontFamily.primary} !important;
+    text-transform: capitalize;
+  }
+
+  svg {
+    width: 68px !important;
+    height: 80px !important;
+    @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+      width: 70px;
+      height: 70px;
+    }
+  }
+
+  .Ministries__ContactImageTitle {
+    font-size: 12px !important;
+  }
+
+  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    width: 100%;
+    padding-left: 20px;
+  }
+`
+
+export const ArticleContactInfo = styled(MinistriesContactInfo)`
+  margin-left: 22px !important;
+`
+
+export const ArticleTitle = styled(Title)`
+  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    font-size: ${(props) => props.theme.fonts.heading1} !important;
+    font-family: ${(props) => props.theme.fontFamily.primary} !important;
+  }
+`
 
 export const PostContainer = styled.div`
   width: 100%;
-  margin-top: 80px;
+  margin-bottom: 100px;
+  margin-top: -15px;
+
+  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    margin-top: -40px;
+  }
 `
 
 export const PostContentContainer = styled.div`
-  display: flex;
-  justify-content: center;
+  @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+    display: flex;
+    justify-content: center;
+  }
 `
 
 export const PostItemsContainer = styled.div`
   img {
     background-size: cover;
     background-position: center center;
+    width: 100%;
+  }
+
+  .content-body {
+    h1 {
+      margin-top: 38px;
+
+      @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+        margin-top: 76px;
+      }
+    }
+
+    h2 {
+      margin-top: 30px;
+      @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+        margin-top: 60px;
+      }
+    }
+
+    img {
+      margin-top: 25px;
+
+      @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+        margin-top: 50px;
+      }
+    }
+
+    p {
+      margin-top: 28px;
+    }
+
+    @media (min-width: ${(props) => props.theme.breakpoints.md}) {
+      p:nth-of-type(2) {
+        position: relative;
+        color: ${(props) => props.theme.colors.secondary};
+
+        &::before {
+          content: '';
+          position: absolute;
+          left: -50px;
+          border-left: 5px solid;
+          height: 100%;
+          width: 10px;
+        }
+      }
+    }
+  }
+
+  p:first-of-type {
+    margin-top: 32px;
+  }
```

Comment:
Need to make sure that the generic blog body style accommodate for all possible inputs from Markdown (Atleast the one that will be used the most so)

h1 to h6, li, ul, ol, p, img and so on. If you dont have the style for them, comment to Phil